src/PhappCommandBase.php

```diff
@@ -83,14 +83,16 @@ protected function initShellEnvironment() {
    *
    * @param string $directory
    *   (optional) The director
+   * @param \drunomics\Phapp\PhappManifest $manifest
+   *   (optional) Manifest
    *
    * @return string[]
    *   The array of environment variables, keyed variable name.
    *
    * @throws \drunomics\Phapp\Exception\PhappEnvironmentUndefinedException
    *   Thrown when the environment is undefined.
    */
-  protected function getPhappEnviromentVariables($directory = './') {
+  protected function getPhappEnviromentVariables($directory = './', $manifest = NULL) {
```

Comment:
What is actually your opinion on having multiple optional parameters? Because in general I am trying to avoid it. Usually it complicates things if you want to set n-th optional argument only. Here it's it's kind of okay-ish, since there are only two of them. Still I would like to hear you opinion.

Reply:
I agree with that in general but find it ok when the order of usage is usually goes in the order of params, e.g. 3 optional param goes with usually optional param 1+2 being set anyway